common-pipelines.yml

```diff
@@ -0,0 +1,38 @@
+jobs:
+- job: ${{ parameters.name }}
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+
+  strategy:
+    matrix:
+      Python27:
+        python.version: '2.7'
+      Python35:
+        python.version: '3.5'
+      Python36:
+        python.version: '3.6'
+      Python37:
+        python.version: '3.7'
+    maxParallel: 4
+
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+      architecture: 'x64'
+   
+  - script: |
+      pip install --upgrade pip
+      pip install pytest
+      pip install -U cookiecutter
+    condition: succeeded()
+    displayName: 'Install cookiecutter'
+  
+  - script: |
+      pytest -s
+    displayName: 'Install IoT Edge Module and Run tests with pytest'
```

Comment:
Suggest `'Scaffold an IoT Edge module and run tests with pytest'`

Reply:
ok